srml/support/src/event.rs

```diff
@@ -452,31 +452,45 @@ macro_rules! impl_outer_event {
 		$( $module_name:ident::Event $( <$generic_param:ident> )? $( { $generic_instance:ident } )?, )*;
 	) => {
 		$crate::paste::item! {
-		// Workaround for https://github.com/rust-lang/rust/issues/26925 . Remove when sorted.
-		#[derive(Clone, PartialEq, Eq, $crate::codec::Encode, $crate::codec::Decode)]
-		#[cfg_attr(feature = "std", derive(Debug))]
-		$(#[$attr])*
-		#[allow(non_camel_case_types)]
-		pub enum $name {
-			system($system::Event),
-			$(
-				[< $module_name $(_ $generic_instance )? >](
-					$module_name::Event < $( $generic_param )? $(, $module_name::$generic_instance )? >
-				),
-			)*
-		}
-		impl From<$system::Event> for $name {
-			fn from(x: $system::Event) -> Self {
-				$name::system(x)
+			// Workaround for https://github.com/rust-lang/rust/issues/26925 . Remove when sorted.
```

Comment:
I know you didn't really add this code, but is this comment actually valid? What exactly is the workaround here?

Reply:
You are right, I removed it :)

Reply:
The comment is so "common" in the Substrate code, that I did not pay attention anymore to it xD